frontend/src/atoms/Paragraph.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export const Paragraph = ({ children, ...props }) => (
+  <p className={classNames(props)}>{children}</p>
```

Comment:
Tohle neni dobre, protoze v props je object, ktery obsahuje vsechny ostatni props krome children. A to je divny posilat do classNames - zvlast, kdyz pak pouzivas jako <Paragraph className={...} ...> - je to vlastne slozitejsi tedka rozsirit tu komponentu a je to matouci pouziti tohle.

- destructuruj jen children, className, ktery pak posles to classNames() v <p> elementu.